src/util.rs

```diff
@@ -284,21 +286,20 @@ pub fn format_message(error_code: u32) -> Result<String, Box<dyn std::error::Err
         )
     };
     if chars_written == 0 {
-        return Err(get_last_error().into());
+        return Ok(get_last_error()?);
     }
     let result = unsafe { buf.to_string()? };
-    if let Err(v) = unsafe { LocalFree(HLOCAL(buf.as_ptr() as *mut _)) } {
-        log::trace!("LocalFree \"{}\"", v);
+    if unsafe { !LocalFree(HLOCAL(buf.as_ptr() as *mut _)).is_invalid() } {
+        log::trace!("LocalFree failed: {:?}", get_last_error());
```

Comment:
```suggestion
        // Win32 returns the same handle if `LocalFree` fails
        log::trace!("LocalFree failed: {:?}", get_last_error());
```
Logic tripped me up the first time since their api is a bit strange